src/views/VenueDetails/VenueDescription/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import Rating from '@mui/material/Rating'
+import Carousel from 'react-material-ui-carousel'
+import styles from './styles.module.css'
+import { VenueImgContainer } from './VenueImgContainer'
+import { Description } from './Description'
+import { Gallery } from './Gallery'
+import { LocationMap } from './LocationMap'
+
+export function VenueDescription ({ venueDetails }) {
+  const [displayDescription, setDisplayDescription] = useState(true)
+  const [displayGallery, setDisplayGallery] = useState(false)
+  const [displayMap, setDisplayMap] = useState(false)
+
+  let imgsToDisplay = venueDetails.venueDescriptionImgUrls
+  imgsToDisplay = { 0: venueDetails.landingImgUrl, ...imgsToDisplay }
+  imgsToDisplay = Object.values(imgsToDisplay)
+
+  const handleDescriptionButton = () => {
+    setDisplayDescription(true)
+    setDisplayGallery(false)
+    setDisplayMap(false)
+  }
+  const handleGalleryButton = () => {
+    setDisplayDescription(false)
+    setDisplayGallery(true)
+    setDisplayMap(false)
+  }
+  const handleMapButton = () => {
+    setDisplayDescription(false)
```

Comment:
What if all of it were true? I know that you got it covered, but try to introduce a datastructure that will not allow such behavior.